Userland/Utilities/sql.cpp

```diff
@@ -304,22 +304,21 @@ class SQLRepl {
                 connect(parts[1]);
             else
                 outln("\033[33;1mUsage: .connect <database name>\033[0m");
-        } else if (command.starts_with(".read ")) {
-            if (!m_input_file) {
-                auto parts = command.split_view(' ');
-                if (parts.size() == 2) {
-                    source_file(parts[1]);
-                } else {
-                    outln("\033[33;1mUsage: .read <sql file>\033[0m");
-                }
-            } else {
-                outln("\033[33;1mCannot recursively read sql files\033[0m");
-            }
+        } else {
+            if (command.starts_with(".read ")) {
+                if (!m_input_file) {
+                    auto parts = command.split_view(' ');
+                    if (parts.size() == 2)
+                        source_file(parts[1]);
+                    else
+                        outln("\033[33;1mUsage: .read <sql file>\033[0m");
+                } else
+                    outln("\033[33;1mCannot recursively read sql files\033[0m");
```

Comment:
Please put braces around both branches, or neither if both are single-line.

Reply:
roger that